### DISCUSSION TITLE
Is it needed `enableStaticRendering()` for every page with SSR?

### DISCUSSION THREAD

**Discussion by kmvan** [mobxjs/mobx#2851] [General] [answered]
```js
enableStaticRendering(typeof window === 'undefined')
```
Not sure where to use `enableStaticRendering`, in `*.tsx` page file or `*.ts` store class file?

**Comment by danielkcz** [comment, marked as answer]
Ideally somewhere on top of your code, some `index.ts` or similar. Doing it on every page might be too late in some cases.

**Reply by kmvan** [reply]
Thanks
